target/postfix/main.cf

```diff
@@ -26,7 +26,10 @@ smtpd_tls_security_level = may
 smtpd_tls_loglevel = 1
 smtp_tls_security_level = may
 smtp_tls_loglevel = 1
+
+# Reduces CPU overhead with `NO_COMPRESSION`, SMTP not at risk of CRIME attack (see git blame for details)
 tls_ssl_options = NO_COMPRESSION
```

Comment:
We could add `NO_RENEGOTIATION` in order to mitigate CPU exhaustion attack. http://www.postfix.org/postconf.5.html#tls_ssl_options

>    Postfix ≥ 3.4. This can reduce opportunities for a potential CPU exhaustion attack.

Reply:
Yes that is a good idea. I'll merge this though, so let's make a separate PR for that.